share/benchmark/xtp_benchmark.in

```diff
@@ -23,9 +23,13 @@ import argparse
 import h5py
 import shutil
 import json
+import math
```

Comment:
Can you please run [autopep8](https://pypi.org/project/autopep8/) over the whole file :) ? You can also do it directly from [VSCode](https://code.visualstudio.com/docs/python/editing#_formatting)

Reply:
ohh yes, how nice. :)